scripts/core/superdesk-api.d.ts

```diff
@@ -903,6 +903,8 @@ declare module 'superdesk-api' {
         saml_auth: any;
         google_auth: any;
         saml_label: any;
+        oidc_auth: any;
+        keycloak_config: any;
```

Comment:
not sure why we need both settings? wouldn't just `keycload_config` do?

Reply:
It's for consistency, because other auth methods have variable format `method`_auth with boolean type to check if it's enabled so we only do the same for OIDC. Should we move `oidc_auth` to `keycloak_config`?

Reply:
makes sense to keep it consistent :+1: